index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/2849147338.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <header>
+        <div class="headone">
+            <div class="headleft">
+                <img src="./asset/shape1.svg" alt="">
+            </div>
+            <div class="headright">
+            <div class="right">
+                <i class="fa-solid fa-bars fa-2x"></i>
+            </div>
+        </div>
+        </div>
+    </header>
+    <main>
+        <section>
+            <h1>Charlie <br> Giorgania</h1>
+            <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. <br> 
+                If you like what you see and have a project you need coded, don’t hesitate and contact me</p>
+            <div class="bot">
+            <input type="button" value="Start Collaboration ">
+        </div>
+
+    </section>
+    <div class="sectiontwo">
+        <!-- <img src="./asset/shape.svg" alt="A joined shape image"> -->
+    </div>
```

Comment:
- [ ] [REQUIRED] Please, make sure to remove commented code because it pollutes the codebase. You can use git to go back to old code.

Reply:
ok i have done it